synapse/tests/test_lib_stormtypes.py

```diff
@@ -530,7 +530,7 @@ def check_fire_mesgs(storm_mesgs, expected_data):
                     ('csv:row', {'row': ['test:str', '9876', '3001/01/01 00:00:00.000'],
                                  'table': 'mytable'}))
 
-            q = 'test:str $lib.csv.emit(:tick, :hehe)'
+            q = 'test:str $hehe=$node.props.hehe $lib.csv.emit(:tick, $hehe)'
```

Comment:
was this purely a demonstrative change?

Reply:
To ensure coverage of the NodeProps deref above :)

Reply:
Oh wait, this was due to the `:hehe` raising an exception when the prop doesn't exist, which is correct behavior for the rest of storm.

Reply:
oh - this is related to the change we discussed this morning